docs/reference/Image.rst

```diff
@@ -76,9 +76,16 @@ Constructing images
 .. autofunction:: new
 .. autofunction:: fromarray
 .. autofunction:: frombytes
-.. autofunction:: fromstring
```

Comment:
I guess it is fair enough to remove this, as there's no docstring to be helpful, the function doesn't do anything, and it's not something we want to advertise.